apps/charge-accounts-service/src/operators/operators.controller.ts

```diff
@@ -7,7 +7,11 @@ import { AuthOperatorDto } from '@app/accounts-service/operators/dto/auth-operat
 import { Response } from 'express'
 import { WebhookEvent } from '@app/apps-service/payments/interfaces/webhook-event.interface'
 import { MessagePattern } from '@nestjs/microservices'
+import { ApiBody, ApiCreatedResponse, ApiNotFoundResponse, ApiOperation, ApiParam, ApiTags, ApiUnauthorizedResponse } from '@nestjs/swagger'
+import { AuthOperator } from '@app/accounts-service/operators/entities/auth-operator.entity'
+import { CreateOperatorUser } from '@app/accounts-service/operators/entities/create-operator-user.entity'
 
+@ApiTags('Operators')
```

Comment:
@LiorAgnin Operators is not available in Swagger UI on port 5002, is it visible to you?

Reply:
Oh is that so. I'll check. Thanks. @MusabShakeel576

Reply:
@MusabShakeel576 The issue is that the Operators are not used in the API service. I've set up a dedicated swagger module for the accounts service to address this. To see it. Just run the backend and go to http://localhost:5001/api#/

Reply:
@LiorAgnin port 5001 is not working, there is no Swagger UI.

Reply:
@MusabShakeel576  I double checked and it works for me. Can you try again?

Reply:
@LiorAgnin sure, I'll try again

Reply:
@LiorAgnin it's working, the issue was, AMPLITUDE_API_KEY env was not set. Maybe we can update the env example file.